api/src/services/queue.service.ts

```diff
@@ -46,15 +46,15 @@ class InstrumentedRabbitQueueService<T> extends InstrumentedComponent implements
   }
 
   public async healthcheck(): Promise<boolean> {
-    return this.instrumentMethod(`queue healthcheck`, async (span: Span) => {
+    return this.instrumentMethod(`${this.messageGroup}.healthCheck send`, SpanKind.INTERNAL, async (span: Span) => {
```

Comment:
For `messaging` the semantic convention is `<destination name> <operation name>`. Operation name could have one of these values: `send`, `receive` or `process`.